package.json

```diff
@@ -33,10 +33,12 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/styles": "^4.10.0",
     "axios": "^0.18.0",
+    "base64url": "^3.0.1",
     "bootstrap": "^4.3.1",
     "core-js": "^2.5.7",
-    "cql-exec-fhir": "^1.5.0-beta.1",
-    "cql-execution": "^2.0.0-beta.1",
+    "cql-exec-fhir": "^1.3.0",
+    "cql-execution": "~1.4.1",
```

Comment:
Did you mean to bump the cel-execution code back to an older version?

Reply:
Nope, I just merged it and selected incoming change every time. Updated.